vos/test/scripts/vospace-move-atest.tcsh

```diff
@@ -112,7 +112,6 @@ $LSCMD $CERT $CONTAINER/e > /dev/null || echo " [FAIL]" && exit -1
 echo " [OK]"
 
 echo -n "check no write permission on source data node (fail)"
-echo "$MVCMD $CERT2 $CONTAINER/b $CONTAINER/d "
```

Comment:
I still hope we'll get to change these scripts to Python. If we could figure out a way to inject the credentials and run them in Travis that would be an incentive.